mungegithub/mungers/submit-queue.go

```diff
@@ -1070,6 +1072,14 @@ func (sq *SubmitQueue) validForMergeExt(obj *github.MungeObject, checkStatus boo
 		}
 	}
 
+	if approvedReview, changesRequestedReview, ok := obj.CollectGHReviewStatus(); !ok {
+		sq.SetMergeStatus(obj, ghReviewStateUnclear)
+		return false
+	} else if len(approvedReview) == 0 || len(changesRequestedReview) > 0 {
```

Comment:
You could split this into two cases to provide a more specific message (i.e. "This PR is missing an 'approved' review" or "This PR has changes requested.")